packages/opensrp-plans/src/ducks/index.ts

```diff
@@ -0,0 +1,341 @@
+import { Dictionary } from '@onaio/utils';
+import intersect from 'fast_array_intersect';
+import { get, keyBy, values } from 'lodash';
+import { AnyAction, Store } from 'redux';
+import { createSelector } from 'reselect';
+import SeamlessImmutable from 'seamless-immutable';
+import { PlanDefinition } from '../plan-global-types';
+import { descendingOrderSort, isPlanDefinitionOfType } from '../helpers/utils';
+
+/** the reducer name */
+export const reducerName = 'PlanDefinition';
+
+/** list of plan types displayed
+ * TODO: To be moved to env
+ */
+export const DISPLAYED_PLAN_TYPES = 'FI,IRS,MDA,MDA-Point,Dynamic-FI,Dynamic-IRS,Dynamic-MDA';
+
+// actions
+
+/** PLAN_DEFINITION_FETCHED action type */
+export const PLAN_DEFINITIONS_FETCHED =
+  'reveal/reducer/opensrp/PlanDefinition/PLAN_DEFINITIONS_FETCHED';
+
+/** PLAN_DEFINITION_FETCHED action type */
+export const REMOVE_PLAN_DEFINITIONS =
+  'reveal/reducer/opensrp/PlanDefinition/REMOVE_PLAN_DEFINITIONS';
+
+/** PLAN_DEFINITION_FETCHED action type */
+export const ADD_PLAN_DEFINITION = 'reveal/reducer/opensrp/PlanDefinition/ADD_PLAN_DEFINITION';
+
+/** interface for plan Objects */
+/** Enum representing the possible intervention types */
+export enum PlanStatus {
+  ACTIVE = 'active',
+  COMPLETE = 'complete',
+  DRAFT = 'draft',
+  RETIRED = 'retired',
+}
+/** Enum representing the possible intervention types */
+export enum InterventionType {
+  DynamicFI = 'Dynamic-FI',
+  DynamicIRS = 'Dynamic-IRS',
+  DynamicMDA = 'Dynamic-MDA',
+  FI = 'FI',
+  IRS = 'IRS',
+  IRSLite = 'IRS-Lite',
+  MDA = 'MDA',
+  MDAPoint = 'MDA-Point',
+  SupplyManagement = 'Supply-Management',
```

Comment:
Is this not supposed to be `SM` instead of `SupplyManagement`?

Reply:
Yeah should be the initials SM = 'SM' as provided here

Reply:
Looks like it should be `SM='Supply-Management'`

Reply:
Actually, you are correct SM='SM'